src/features/classification/security-controls/SecurityControls.js

```diff
@@ -28,7 +29,8 @@ const SecurityControls = ({ controls, controlsFormat, maxAppCount }: Props) => {
         const shortMessage = getShortSecurityControlsMessage(controls);
         items = shortMessage ? [shortMessage] : [];
 
-        if (items.length && controlsFormat === SHORT_WITH_TOOLTIP) {
+        // will remove SHORT_WITH_TOOLTIP after eua release
```

Comment:
I would add a `TODO` to this so most IDEs will call out this needs to be removed.

Reply:
Thanks for reminding! I will remove it in my following pr.